Documentation/Wireless.md

```diff
@@ -0,0 +1,443 @@
+
+
+# Wireless Assembly
+
+*How to build an **IOT Hotdog**.*
+
+<br>
+
+## Parts list
+
+<img
+  src = '../Resources/Wireless/parts.jpeg'
+  width = 420
+/>
+
+<br>
+
+### Custom Parts
+
+#### 3D Print
+
+- **[Top Casing]**
+
+- **[Bottom Casing]**
+
+<br>
+
+#### Order PCBs
+
+Estimated Cost : `10 - 20 US$` per PCB
+
+- **[Front PCB]**
+
+- **[Back PCB]**
+
+###### Specifications:
+
+- Solder Mask : `Yes`
+- Thickness : `1.6mm`
+- Layers: `2`
+- Size : `100 x 50 mm`
+- HASL : `Yes`
+
+<br>
+
+### Stock Parts
+
+- **`A1` : Arduino Nano or compatible + two `1x15` pin female headers**
+
+- `BT1` : AAA battery holder (thru-hole mount recommended)
+
+- `C1` : `10 - 100uF` capacitor (optional)
+
+- `C2` : `~1000uF` capacitor (optional)
+
+- **`D1-D3` : three white LEDs, I used `334-15` / `T1C1-4WYA`**
+
+- **`12cm` long hook-up wire connecting `J1` to `J4`**
+
+- **`12cm` long hook-up wire connecting `J2` to `J3`**
+
+- **`R1-R5` : five GL5528 photoresistors**
+
+- **`R6` : 1kΩ resistor (or lower if you're using less bright LEDs)**
+
+- `SW1` : **SPDT** switch
+
+- `U1` : `nRF24L01` + a `2x4` pin female header
+
+- `U2` : `0.8 - 5V` to `5V` boost converter + a `1x3` pin female header
+
+- A transparent sleeve, I used **[This][Sleeve]** one.
+
+- Mini zip ties
+
+- Rubber bands
+
+<br>
+<br>
+
+## Programming
+
+### Arduino Nano
+
+Extract this **[Zip]** file and upload <br>
+the file `hotdog-fw.ino.hex` to <br>
+the Nano using **[XLoader]**.
+
+<img
+  src = '../Resources/Wireless/xloader.png'
+  width = 210
+/>
+
+##### Windows
+
+By default you should be able to <br>
+use `COM0`, if not try other ports.
+
+<br>
+
+### Other 
+
+*Nano-compatible microcontrollers*
+
+Compile & upload the **[Firmware]** using the **[Arduino IDE]**.
+
+##### Dependencies
+
+- **Protothreads**
+- **BTLE**
+- **RF24**
+
+<br>
+<br>
+
+## Assembly
+
+### Soldering Font PCB
+
+1.  Place the `1x15` headers on the Nano.
+
+    *I didn't have 1x15, so I cheated and yanked* <br>
+    *out one of the terminals from a `1x16`.*
+    
+    *Don't be like me.*
+
+    <img
+      src = '../Resources/Wireless/A1_prep.jpeg'
+      width = 210

```

Comment:
Because it looked less organized without it.

The images aren't all of the same size, as such image pairs
that use the same width instead of the same height can look
rather randomly placed.

If you mean that singular images should be as tall 
as the image pairs then I'd agree with you.

Reply:
I adjusted the images to be relatively 
sized to their neighbours.

Reply:
Thank you, looks much better.